extensions/amp-sidebar/amp-sidebar.md

```diff
@@ -0,0 +1,93 @@
+<!---
+Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# <a name="amp-sidebar"></a> `amp-sidebar`
+
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>A sidebar provides a way to display meta content intended for temporary access (navigation links, buttons, menus, etc.).The sidebar can be revealed by a button tap while the main content remains visually underneath.</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Availability</strong></td>
+    <td><a href="https://www.ampproject.org/docs/reference/experimental.html">Experimental</a>; no validations yet.</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Required Script</strong></td>
+    <td><code>&lt;script async custom-element="amp-sidebar" src="https://cdn.ampproject.org/v0/amp-sidebar-0.1.js">&lt;/script></code></td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Examples</strong></td>
+    <td>None</td>
+  </tr>
+</table>
+
+## Behavior
+
+- There can be only one `<amp-sidebar>` in an AMP document. The `<amp-sidebar>` should be a direct child of the `<body>`.
+- The sidebar can only appear on the left or right side of a page.
+- The `<amp-sidebar>` may contain any valid HTML elements (supported by AMP).
+- The `<amp-sidebar>` may not contain any `builtins` or `extensions` except for `amp-accordion`, `amp-img` and `amp-fit-text`
+- The max-height of the sidebar is 100vh, if the height exceeds 100vh then a vertical scrollbar appears. The default height is set to 100vw in CSS and is overridable in CSS.
+- The default width of the sidebar is 30vw and the max-width is 80vw and the width can be adjusted using CSS.
```

Comment:
Not according to code. The width is between `30vw` and `80vw`, but it will take width according to its dynamic layout.